loops.js

```diff
@@ -17,8 +21,14 @@
 // loop...
 // klaar!
  // ==========================================
-
-
+for (let i = 0; i<5; i++){
+    if(i <= 3){
+        console.log('loop...');
+    }
+    if(i >= 4){
+        console.log('hoera!')
+    }
+}
 
```

Comment:
for (let i = 0; i < 5; i++) {
    if (i < 4) {
        console.log("loop")
    } else {
        console.log("klaar")
    }
}

je kan 'else' gebruiken voor de rest.